### PR TITLE
Implement Strawberry

### PR DESCRIPTION
**Pull request by AndreasPB**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -37,7 +40,23 @@ services:
...
+        container_name: graph_db
+        volumes:
+            - ./conf:/conf
+            - ./data:/data
+            - ./import:/import
+            - ./logs:/logs
+            - ./plugins:/plugins
+        environment:
+            - NEO4J_AUTH=none
+            - NEO4J_dbms_security_auth__enabled=false
```
Comment (Primdahl26) [inline docker-compose.yml]: Lets not spin up `neo4j` when we use `sqlalchemy` anyway with `strawberry`?

**Review by Primdahl26 — APPROVED** [review]
In the graphql `v1` endpoint `getUser`, `getPerson` & `getCountry` does not work 

```
{
  "data": null,
  "errors": [
    {
      "message": "Field 'getCountry' argument 'id' of type 'Int!' is required, but it was not provided.",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": null
    }
  ]
}
```

**Comment by AndreasPB** [comment]
#5

File: docker-compose.yml (hunk 1 above)
Comment (AndreasPB) [reply docker-compose.yml]: Yy, removed

**Pull request merged**